.github/workflows/ca-ds-connection-test.yml

```diff
@@ -163,8 +165,10 @@ jobs:
           sleep 60
 
           # enrollment should work
+          # OCSP is not working because he authorityMonitor is disabled
```

Comment:
I think this is another issue with `AuthorityMonitor`, basically if cannot be disabled without causing issues somewhere elase.